relm4/src/typed_view/column.rs

```diff
@@ -238,6 +248,9 @@ where
         let sort_fn = C::sort_fn();
 
         let c = gtk::ColumnViewColumn::new(Some(C::COLUMN_NAME), Some(factory));
+        c.set_resizable(C::ENABLE_RESIZE);
+        c.set_expand(C::ENABLE_EXPAND);
+        c.set_resizable(C::ENABLE_RESIZE);
```

Comment:
Is there a reason why `set_resizable` is called twice?

Reply:
That's a mistake on my part, let me fix it now. Thanks for pointing it out.